samples/Build2019Demo.Unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -627,21 +627,24 @@ PlayerSettings:
   webGLLinkerTarget: 1
   webGLThreadsSupport: 0
   scriptingDefineSymbols:
-    1: SPATIALALIGNMENT_ASA;STATESYNC_TEXTMESHPRO
```

Comment:
Are there any drawbacks with compiling the TextMeshPro bits unconditionally?

Reply:
1) It introduces a dependency on the text mesh pro unity package.
2) Folks consuming a different version of text mesh pro may have problems.

It's a bit unclear right now who uses Text Mesh Pro vs who doesn't. I think a more viable solution would be to move these classes into a separate dll with the text mesh pro dependency if folks voice that they really can't consume the Text Mesh Pro dependency.

I also think that the regressions caused by having these preprocessor definitions and the lack of discoverability around enabling/disabling these preprocessor definitions advocates for not keeping them.

Reply:
technically right now the spectator view asmdef already has a text mesh pro dependency, so this preprocessor directive isn't helping avoid a text mesh pro dependency.